Mezzurite.Angular/README.md

```diff
@@ -33,6 +33,7 @@
 ### Installation
 Install mezzurite from npm:
 ```javascript
+  npm install "@microsoft/mezzurite-core"
```

Comment:
this isn't needed because it's automatically installed when you install mezzurite-angular.

Reply:
Done!